### ISSUE TITLE
Add react-native@0.71.x support

### ISSUE THREAD

**Issue by retyui** [mrousavy/react-native-mmkv#496]
Android build fail on `react-native@0.71.x`

```java
npx react-native init RN71 --version 0.71.0-rc.5
cd RN71
yarn add react-native-mmkv
yarn android


FAILURE: Build failed with an exception.

* Where:
Build file '/home/i/open-source/RN71/node_modules/react-native-mmkv/android/build.gradle' line: 173

* What went wrong:
A problem occurred evaluating project ':react-native-mmkv'.
> Expected directory '/home/i/open-source/RN71/node_modules/react-native/android' to contain exactly one file, however, it contains no files.
```

**Comment by whalemare** [comment]
For authors of `react-native-mmkv`

Because of this: https://github.com/facebook/react-native/tree/main/android

**Comment by hamdij0maa** [comment]
same issue

**Comment by hengkx** [comment]
any progress

**Comment by retyui** [comment]
you can use an early version from git
```tsx
yarn add mrousavy/react-native-mmkv#feat/migrate-rn-71
```

> **Require** to enable `newArchEnabled=true` in `android/gradle.properties` file

**Comment by sleaper** [comment]
> 

Does not work for me. I can't download the early version.

**Comment by mym0404** [comment]
`2.6.1` should fix this.

**Comment by mrousavy** [comment]
Fixed by #480 (released in 2.6.1.) :)

**Comment by mrousavy** [comment]
> Thank you so much to everyone working on this repo ❤️

🖤
